internal/v1/api.yaml

```diff
@@ -106,6 +106,17 @@ paths:
             default: 0
             minimum: 0
           description: composes page offset, default 0
+        - in: query
+          name: ignoreImageTypes
+          # ['rhel-edge-installer', 'rhel-edge-commit', ...]
```

Comment:
Remove the comment and maybe add it as an example.

Reply:
added